src/mode.js

```diff
@@ -59,11 +59,13 @@ function getMode_() {
       // occur during local dev.
       !!document.querySelector('script[src*="/dist/"],script[src*="/base/"]');
 
-  const overrideDevelopment = parseQueryString(location.hash)['development'];
+  const overrideDevelopment = parseQueryString(
+      // location.originalHash is set by the viewer when it removes the fragment
```

Comment:
FYI, Viewer also remembers all original args in hash and can return them via `getParam(name)`. You can always use this if convenient/possible.